### ISSUE TITLE
Create new window keyboard shortcut Ctrl+Shift+N does not work in Windows 10 on latest VS Code Insiders build

### ISSUE THREAD

**Issue by altosaar** [microsoft/vscode#124271]
Does this issue occur when all extensions are disabled?: Yes

Version: 1.57.0-insider (system setup)
Commit: 53b58c89d44deb230dc526605561a1d42f6d1c50
Date: 2021-05-19T13:57:29.889Z
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19042

Steps to Reproduce:

1. Install latest VS Code Insiders from https://community.chocolatey.org/packages/vscode-insiders
2. Open VS Code
3. Try to create a new window with the keyboard shortcut Ctrl+Shift+N
4. The window does not open

**Comment by bpasero** [comment]
I cannot reproduce. 

Can you try to run `code --disable-extensions --user-data-dir <directory>` where `<directory>` is an empty folder? This will ensure Code is starting with a fresh data directory, e.g. no specific settings and without any extensions running.

**Comment by altosaar** [comment]
Thanks @bpasero ! I wonder what is quickest to debug if you cannot reproduce. 

I just tried running `code --disable-extensions --user-data-dir tmp` where `tmp` is empty, and am still seeing the same behavior:

![image](https://user-images.githubusercontent.com/5317244/119139000-88112480-ba10-11eb-80d0-0e62d02d88fa.png)

Maybe there is a way to check the logs of keypress events in windows?

**Comment by bpasero** [comment]
When you open the keybindings editor and search for any key assigned to `Ctrl+Shift+N`?

**Comment by altosaar** [comment]
Thanks! Yes it looks like it's mapped correctly:

![image](https://user-images.githubusercontent.com/5317244/119162208-1e9d1000-ba28-11eb-9c6f-25a56c29b9ba.png)

**Comment by bpasero** [comment]
And File > New Window works?

**Comment by altosaar** [comment]
Yes, that's what I've been using :) but I am working on VS Code accessibility features (e.g. voice navigation) that require keyboard shortcuts, so it would be very helpful to figure out where I'm going wrong here

**Comment by bpasero** [comment]
Does this reproduce when you first close all editors and then hit `Ctrl+Shift+N`, and if so, is there anything printed to the devtools console:

Please see if you can find any logged output that could help us find the source of the issue:
* run `code --verbose` from the command line and attach the output
* in Code select **Help | Toggle Developer Tools** and select the **Console**
* attach any output you see in the **Console** you can find

Thanks!

**Comment by altosaar** [comment]
Hmm, seems like this was a Windows bug. 

Fixed! 

I realized that `Ctrl+Shift+N` was not working in Chrome either (for New Incognito Window). 

I remapped Ctrl(left) + shift(right) + N  to Ctrl(right) + Shift(left) + N using the PowerToys keyboard manager https://docs.microsoft.com/en-us/windows/powertoys/keyboard-manager

Then I restarted.

This seems to have fixed it - Ctrl+Shift+N works in VS Code and Chrome. 

It could be a weird side effect from using Bootcamp as well. Ctrl+Shift+N works even when I disable PowerToys or the keyboard manager. 

Thanks so much for sticking with this!!! 

Will update if anything comes up again - hope this helps anyone else on Bootcamp / Windows who finds `Ctrl+Shift+N` or the New Incognito Window or New Window keybindings / keyboard shortcuts aren't working :)
